README.md

```diff
@@ -1,10 +1,13 @@
 # findutils
 
 [![Crates.io](https://img.shields.io/crates/v/findutils.svg)](https://crates.io/crates/findutils)
+[![Discord](https://img.shields.io/badge/discord-join-7289DA.svg?logo=discord&longCache=true&style=flat)](https://discord.gg/wQVJbvJ)
+[![License](http://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/uutils/findutils/blob/main/LICENSE)
 [![dependency status](https://deps.rs/repo/github/uutils/findutils/status.svg)](https://deps.rs/repo/github/uutils/findutils)
 [![codecov](https://codecov.io/gh/uutils/findutils/branch/master/graph/badge.svg)](https://codecov.io/gh/uutils/findutils)
 
-Rust implementation of [GNU findutils](https://www.gnu.org/software/findutils/).
+Rust implementation of [GNU findutils](https://www.gnu.org/software/findutils/) and xargs.
```

Comment:
Isn't "and xargs" redundant because `xargs` is part of GNU `findutils`?

Reply:
We can just list them all, I think. It's only 4 utils:
```
Rust implemementation of GNU findutils: `xargs`, `find`, `locate` and `updatedb`.